apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConstants.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2020 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Spanner.Data
+{
+    /// <summary>
+    /// Constants for use in Spanner.
+    /// </summary>
+    internal class SpannerConstants
```

Comment:
We really don't have this kind of file in any of the libraries. We just put the constants where they are being used/makes more sense, and in this case the clear winner is the `InstanceHostManager` class. Sorry if I wasn't clear when I said to have the name of the variable as a constant, I meant on the same `InstanceHostManager` class not on a separate `SpannerConstants` class. The property that checks whether the variable is set can be moved there as well.